riscv-ffh.adoc

```diff
@@ -4,9 +4,7 @@
 :company: RISC-V.org
 // Pull in variable with revision info
 include::autogenerated/revision.adoc-snippet[]
-:revnumber: 1234
-:revdate: 2023-01-01
-:revremark: Draft
+:revremark: This document is stable. Assume anything could still change, but limited change should be expected. Visit http://riscv.org/spec-state for further details.
```

Comment:
Let's use a separate patch to switch the state from draft to stable.

Reply:
Sure. This PR was just for Rafel to show the CI warnings I was getting. I will cleanup once his PR is merged.